com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/validators/parameters/RoutesValidator.java

```diff
@@ -29,7 +29,7 @@ public RoutesValidator() {
     public boolean isValid(Object routes) {
         List<Map<String, Object>> routesList = applyRoutesType(routes);
 
-        if (routesList == null) {
+        if (routesList.isEmpty()) {
```

Comment:
You can use CollectionUtils.isEmpty(routesList) just to be null safe. I see that empty list is returned but in case of future changes.

Reply:
done